services/sidecar/src/simcore_service_sidecar/mpi_lock.py

```diff
@@ -78,12 +63,30 @@ async def try_to_acquire_lock(
     return None
 
 
-async def acquire_lock(cpu_count: int) -> bool:
+async def wrapped_acquire_and_extend_lock_worker(
+    reply_queue: multiprocessing.Queue, cpu_count: int
+) -> None:
+    try:
+        # if the lock is acquired the above function will block here
+        await acquire_and_extend_lock_forever(reply_queue, cpu_count)
+    except Exception:  # pylint: disable=broad-except
+        logger.exception("An exception ocurred while trying to acquire the mpi lock")
+        reply_queue.put(False)
```

Comment:
MINOR: there is not need for this ``reploy_queue.put`` here since the one in ``finally:`` will always execute

Reply:
yes, you are correct